lessons/helper/py/sprint_one.yaml

```
@@ -0,0 +1,60 @@
+node_ref: "helper.py.sprint_one"
+content:
+  - type: "paragraph"
+    value: |
+      # Welcome to the first Sprint of our Python Course
+      Congratulations on taking the first steps in your computer science journey! 
+      Whether you want to start a new career and become a programmer or add a skill to your toolbelt for parsing data like a pro, 
+      we've got you covered with our Python Introduction Course. We will have a great month together where we will get our hands 
+      dirty with challenging projects, learn about new ideas and engage with a fantastic community of like-minded peers.
+
+      ## What awaits us in the course?
+      This course is intended for absolute beginners in programming or computer science but is not limited to them. 
+      Thanks to our adaptable roadmap, you can customise your learning journey by exploring sibling lessons that 
+      provide more information on complex concepts or uncover the subject's inner workings. We will focus on learning syntax, 
+      basic operations, functions and data types in Python and touch on how programming language interacts with an 
+      operating system (by reading and writing files). We will explore concepts like Object Oriented Programming and 
+      learn about essential developer tools and platforms like Vscode, GitHub and Git.
+
+      The goal is to give you a good starting ground for learning new skills and challenges. It could be Data Science,
+      Backend Development, System Administration, DevOps or even Desktop App development. 
+      The beauty of Python is its simplicity and versatility because it's used in many areas of the Computer Science industry.
+
+  - type: "paragraph"
+    value: |
+      ## How will we learn?
+      Our learning journey will last for 1 month and have 2 Sprints. 
+      Each Sprint will be 2 weeks long and have a Project at the end of it. 
+      The Project at the end of the second Sprint will be reviewed by an actual software engineer. 
+      The process will be similar to how it goes in the best tech companies; 
+      that is how all software engineers learn and improve their craft. 
+
+      As for the lessons, they are already available in your roadmap for this course. You will notice that sometimes more 
+      than one lesson is open. That is because you can choose what and when to learn! There is still some order due to 
+      dependency on previous knowledge, but within those parallel lessons, you can go in any order.
```

Parallel lessons кажется такой фразы нет в английском. Нужно чем-то заменить
```suggestion
      dependencies between concepts. But within those parallel lessons you can go in any order.
```